plugins/lang-python/src/test/java/org/elasticsearch/script/python/PythonSecurityTests.java

```diff
@@ -66,12 +68,12 @@ private void assertFailure(String script) {
             doTest(script);
             fail("did not get expected exception");
         } catch (PyException expected) {
-            Throwable cause = expected.getCause();
             // TODO: fix jython localization bugs: https://github.com/elastic/elasticsearch/issues/13967
-            // this is the correct assert:
-            // assertNotNull("null cause for exception: " + expected, cause);
-            assertNotNull("null cause for exception", cause);
-            assertTrue("unexpected exception: " + cause, cause instanceof SecurityException);
+            // we do a gross hack for now
```

Comment:
Can you expand on this comment to explain how this "gross hack" works? I don't get it at a glance...